### DISCUSSION TITLE
How to group multiple columns in a data table into one object?

### DISCUSSION THREAD

**Discussion by BiosSun** [prisma/prisma#8464] [Q&A] [answered]
For example, I have the following model:

```prisma
model Book {
    id     Int        @id
    title  String
    
    ratingScore   Float
    ratingVotes   Int
    ratingStars1  Int
    ratingStars2  Int
    ratingStars3  Int
    ratingStars4  Int
    ratingStars5  Int
}
```

But I want to get the following data structure:

```typescript
interface Book {
    id: number;
    title: string;

    rating: {
        score: number;
        votes: number;
        stars1: number;
        stars2: number;
        stars3: number;
        stars4: number;
        stars5: number;
    }
}
```

What should I do?

**Comment by ryands17** [comment, marked as answer]
@BiosSun 👋 

You would need to fetch the data first and then transform it in your application itself.

**Reply by BiosSun** [reply]
Oh, OK, thank you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
